src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -8,10 +8,10 @@ $header-height: 45px;
   position: fixed;
   display: flex;
   flex-flow: column;
-  top: 66px;
+  top: $navbar-height;
   right: 0;
   width: $chat-sidebar-width;
-  height: calc(100vh - 96px);
+  height: calc(100vh - #{$navbar-height} - #{$footer-height});
```

Comment:
Would be nice if it is in line with 

```ts
$header-height: 45px;
$reserved-page-height: $navbar-height + $footer-height;
```
from https://github.com/sparkletown/sparkle/pull/2007/files

Reply:
fixed 61d9df7